### PR TITLE
Improve appearance and behavior of ReST `:download:` links

### PR DESCRIPTION
**Pull request by Andy-Grigg**
This is a relatively minor PR to improve how links generated with the ReST `:download:` directive work:

- CSS to theme them as hyperlinks instead of code blocks.
- JS to set them to open in a new tab by default. In my use case, this means download .py / .ipynb files instead of opening them in the current tab

Before / after screenshots:

image

image

### CODE REVIEW

File: pyansys_sphinx_theme/static/js/download_target_blank.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/* Add target="_blank" attribute to all hyperlinks generated by the ReST :download: directive.
+* This will ensure the links open in a new tab. */
+
+$(document).ready(function () {
```
Comment (germa89) [inline pyansys_sphinx_theme/static/js/download_target_blank.js:4]: Very clever!

**Review by germa89 — APPROVED** [review]
I think it is good. This is a very good idea and the result looks super nice. Honestly I'm not the most appropriate to judge your work, but I like it!

**Comment by Andy-Grigg** [comment]
> I think it is good. This is a very good idea and the result looks super nice. Honestly I'm not the most appropriate to judge your work, but I like it!

Thanks! Do you want me to hold off until @akaszynski has a chance to review, or shall I merge?

**Comment by germa89** [comment]
Up to you really, if you are in rush, just merge it. If not, surely @akaszynski  might have some useful contributions.

**Comment by akaszynski** [comment]
Nothing to add. Thanks!

**Pull request merged**